client/src/pages/ProfilePhoto/useStyles.ts

```diff
@@ -0,0 +1,40 @@
+import { makeStyles } from '@material-ui/core/styles';
+const useStyles = makeStyles(() => ({
+  paper: {
+    margin: 'auto',
+  },
+  main: {
+    background: 'primary',
+    margin: ' 20vh auto',
+    width: '60vw',
+  },
+  box: {
```

Comment:
These styles can be applied directly to the `Box` component above

Eg: `<Box display="flex" flexDirection="column" padding={10} height={750}>`